server/service/src/sync/README.md

```diff
@@ -1,3 +1,99 @@
+# Sync - Syncrhonisation
+
+Open mSupply is often used in environments with intermittent and low bandwidth internet and therefore needs to be accessible in offline mode. 
+A mechanism that allows for this to happen is called **Sync** (or **Syncrhonisation**).
+
+## Glossary
+
+`site`: An instance of Open mSupply app with one or more active store
+
+`remote site`: This is the same as a site, but we quite often refer to it as ‘remote site’ to emphasise that it’s not always accessible via the internet, and even though it’s part of the same system, it’s not always directly connected to the system.
+
+`central server`/`configuration server`: The main site of mSupply system, this is where central data is configured, including credentials for remote sites, users and active store associations with sites. This is also the server that ‘drives’ synchronisation, all remote sites will synchronise with the central server, which in turn can forward records to other sites (for example, in case of transfer).
+
+`active store`: A store can only belong to one site, even though data for any store is accessible on the central server, certain remote records that are owned by a store can only be edited on a site where the store is active. 
+
+`transfer`: Some data needs to be transferred between sites, for example a shipment from one site to another, we call these records transfers. See [processors](https://github.com/msupply-foundation/open-msupply/blob/bc83acbb3cd51fe3375ac01135c6eb880a793936/server/service/src/processors/README.md#L1) for more details
+
+## Record Types
+
+Records are treated differently by central server and remote sites based on data types, for example remote records belong to only one store and are only editable on a site where this store is active, whereas central data is only editable on a central server, the former usually syncs from remote site to central server whereas the later will only travel from central server to remote site.
+
+`remote record/data`: Can only be owned and edited by one store, on a site where this store is active, this data typically travels from remote site where it’s created and edited, to central server, which in turn may decide to forward it to another remote site (in case of transfers). Shipment, requisition, stock_line are examples of remote record.
+ 
+`central record/data`: This record is only editable on central server, and only travels in one direction, from central server to all remote sites. All central data will sync to all remote sites. Items and units are examples of central records.
+
+`central-remote record/data`: Similar in editability to central data, but will only sync to remote sites where these records are relevant. For example, master_list or name_store_join.
```

Comment:
```suggestion
`central-remote record/data`: Similar in edibility to central data, but will only sync to remote sites where these records are relevant. For example, master_list or name_store_join.
```

Reply:
@mark-prins, hmm i guess editability is not technically a word ? But i've seen it used in places ¯\_(ツ)_/¯ 
sorry I am struggling to ingest the suggestion : )

Reply:
Might want to rephrase it altogether since edibility isn't the correct word for this either 😆 

Though... You can eat data (:

Reply:
confused face: `edibility` is a word. It's just spelt edibility and not editability for some reason 🤷

Reply:
`Definitions of edibility. the property of being fit to eat. synonyms: edibleness. types: digestibility, digestibleness. the property of being easy to digest.`

Reply:
oops

Reply:
ok, `These records are editable in the same way as central data, but it will only sync to remote sites where these records are relevant. For example, master_list or name_store_join.`

Reply:
Thanks: 202442a2e00142e979213c6e4941a33e2f8ec7bb